### PR TITLE
remove is_pyqt5()

### PR DESCRIPTION
**Pull request by AI-Pranto**
QtCore.qVersion() instead of is_pyqt5 function

### CODE REVIEW

**Comment by pshriwise** [comment]
Can you provide some motivation for this change? Was there a problem with the way it was before?

**Comment by AI-Pranto** [comment]
Not a big issue, just want to remove the warning message. Is it ok @pshriwise?
```
The is_pyqt5 function was deprecated in Matplotlib 3.3 and will be removed two minor releases later. Use QtCore.qVersion() instead.
  if is_pyqt5():
```

**Comment by pshriwise** [comment]
Ah, yeah that definitely makes sense. I hadn't seen this in matplotlib yet.

File: openmc_plotter/plotgui.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
 from .scientific_spin_box import ScientificDoubleSpinBox
 from .custom_widgets import HorizontalLine
 
-if is_pyqt5():
+if QtCore.qVersion():
     from matplotlib.backends.backend_qt5agg import FigureCanvas
 else:
     from matplotlib.backends.backend_qt5agg import FigureCanvas
```
Comment (pshriwise) [inline openmc_plotter/plotgui.py:25]: Looking at this again, we can probably just remove this check. The lines in under the `if` and `else` are exactly the same...

```suggestion
from matplotlib.backends.backend_qt5agg import FigureCanvas
```

**Comment by pshriwise** [comment]
Thanks @AI-Pranto!

**Pull request merged**